library/Mockery/Mock.php

```diff
@@ -740,9 +743,7 @@ public function mockery_returnValueForMethod($name)
                 return null;
 
             case 'object':
-                if (\PHP_VERSION_ID >= 70200) {
```

Comment:
This is safe to do because in PHP less than 7.2, `object` will be parsed to `\object` by my new code.